tests/integration/cli/action/test_action_classification.py

```diff
@@ -108,6 +108,8 @@ def test_otx_export_onnx(self, template, tmp_dir_path):
     @pytest.mark.skipif(TT_STABILITY_TESTS, reason="This is TT_STABILITY_TESTS")
     @pytest.mark.parametrize("template", templates, ids=templates_ids)
     def test_otx_eval_openvino(self, template, tmp_dir_path):
+        if template.model_template_id == "Custom_Action_Classification_MoViNet":
+            pytest.xfail("CVS-112824: MoViNet inference fails in OV 2023.0")
```

Comment:
Same here

Reply:
Updated, we already have an issue on github